docs/secret.rst

```diff
@@ -81,7 +81,7 @@ reason, many protocols derive a new key for each session, reset the counter
 to zero with each new key, and never store the derived key or the counter.
 
 You can safely generate random nonces by calling
-:class:``~nacl.utils.random(SecretBox.NONCE_SIZE)``.
+:class:`~nacl.utils.random(SecretBox.NONCE_SIZE)`.
```

Comment:
That's a fair point, d'oh! Since `nacl.utils.random` is used a few places in the documentation, should this be a documented function in `docs/utils.rst` that we can link to from here?